Plate_2_data/4.processing_features/processing_features_plate2.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# initialize the correct shell for your machine to allow conda to work (see README for note on shell names)
+conda init bash
+# activate the main conda environment
+conda activate interstellar_data
+
+# convert all notebooks to python files into the scripts folder
+jupyter nbconvert --to python --output-dir=scripts/ *.ipynb
+
+# run the python scripts in order (from convert+merge, annotate, normalize, and feature select)
+python scripts/merge_sc_plate2.py && python scripts/annotate_sc_plate2.py && python scripts/normalize_sc_plate2.py && python scripts/feature_select_sc_plate2.py
```

Comment:
It would work, but I was hoping to continue with the one line command pattern.

This is definitely a preference as I use that command above for my other modules since I normally only have one notebook. I think I will stick with this but I appreciate the perspective on readability since it is a pretty long line.